src-electron/generator/matter/darwin/Framework/CHIP/templates/helper.js

```diff
@@ -240,9 +240,10 @@ async function asObjectiveCClass(type, cluster, options) {
   }
 
   if (isStruct) {
-    const structObj = await zclQuery.selectStructByName(
+    const structObj = await zclQuery.selectStructByNameAndClusterName(
       this.global.db,
       type,
+      cluster,
       pkgIds
     );
     if (structObj.structClusterCount == 0) {
```

Comment:
So when I try this I get:
```
TypeError: Cannot read properties of undefined (reading 'structClusterCount')
    at Object.asObjectiveCClass (/Users/bzbarsky/zap/dist/src-electron/generator/matter/darwin/Framework/CHIP/templates/helper.js:203:23)
    at Object.asObjectiveCType (/Users/bzbarsky/zap/dist/src-electron/generator/matter/darwin/Framework/CHIP/templates/helper.js:238:19)
```

because `selectStructByName` populates that by the "by name and ..." APIs do not.